packages/web/docs/src/app/layout.tsx

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { ReactNode } from 'react';
+import localFont from 'next/font/local';
+import { Layout, useConfig } from 'nextra-theme-docs';
```

Comment:
it will be better to use `<GuildLayout>` from `@theguild/components`, so we can polish this layout to use across all websites

Reply:
Yeah, I started from that, but needed to iterate on it here to make the feedback loop shorter. Might add this to Components as a new layout or update GuildLayout, but I didn't wanna do this while you were OOO.

I ejected from `GuildLayout` to provide `navbar` and `footer` as components instead of `navbarProps` (Generally I'm more of a fan of compound components instead of spreading props objects.), needed to work with `logo` which would require `layoutProps` and `pageMap` that can't be overwritten.

<img width="721" alt="image" src="https://github.com/user-attachments/assets/5a9c4244-e8d2-4565-9fbf-8b7a216067cc">